source/common/upstream/outlier_detection_impl.cc

```diff
@@ -471,6 +475,20 @@ void DetectorImpl::ejectHost(HostSharedPtr host,
           runtime_.snapshot().getInteger(BaseEjectionTimeMsRuntime, config_.baseEjectionTimeMs()));
       const std::chrono::milliseconds max_eject_time = std::chrono::milliseconds(
           runtime_.snapshot().getInteger(MaxEjectionTimeMsRuntime, config_.maxEjectionTimeMs()));
+
+      // Generate random jitter so that not all hosts uneject at the same time,
+      // which could possibly generate a connection storm.
+
+      // Retrieve max_eject_time_jitter configuration and then calculate the jitter.
+      const int64_t max_eject_time_jitter = runtime_.snapshot().getInteger(
```

Comment:
nit: `getInteger()` returns an unsigned value.
```suggestion
      const uint64_t max_eject_time_jitter = runtime_.snapshot().getInteger(
```